### ISSUE TITLE
[Support]: Get Nvidia to work on all my Cameras

### ISSUE THREAD

**Issue by Sarelseekat** [blakeblackshear/frigate#2373]
### Describe the problem you are having

I've migrated my Frigate deployment from a HASS.io to the suggested bare metal linux system running Frigate in docker. After a day's struggle, I got everything working. As in I got as far as getting the Cuda Toolkit loaded and Nvidia working on two cameras. The rest does not want to work no matter what input_args is use and what is strange is that the two cameras that work, only use - -c:v and - h264_cuvid. No additional input args work.

I'm stuck at this point so I removed the input_args on all the other cameras and not they run through the CPU. I would really love if someone can help me to get all my camera's to run through Nvidea, which is the main reason I made the switch.

### Version

0.9.4-26AE608

### Frigate config file

```yaml
This is the one that works

  Klaasenbosch:
    ffmpeg:
      input_args: 
        - -c:v
        - h264_cuvid
      inputs:
        - path: rtsp://.................:554/Streaming/channels/101
          roles:
            - detect
            - rtmp
    live:
      height: 720
      quality: 20
    detect:
      width: 1920
      height: 1080
    record:
      enabled: True
      retain_days: 1
      events:
        max_seconds: 300
        pre_capture: 5
        post_capture: 5
        retain:
          objects:
            default: 14
    rtmp:
      enabled: True
    objects:
      track:
        - person
        - motorcycle
        - bicycle
        - car
    snapshots:
      enabled: True
      timestamp: False
      bounding_box: False
      retain:
        default: 7
    motion:
      mask:
      - 1920,0,0,0,0,616,165,610,397,603,531,557,637,551,713,520,742,482,771,470,815,502,867,511,913,531,1017,550,1048,551,1110,598,1216,681,1522,658,1920,591
      - 0,1080,1920,1080,1920,781,0,778


This is one that doesn't


  Sondal&Eversdal:
    ffmpeg:
      inputs:
        - path: rtsp://..................:5554/Streaming/channels/101
          roles:
            - detect
            - rtmp
    live:
      height: 720
      quality: 20
    detect:
      width: 1920
      height: 1080
    record:
      enabled: True
      retain_days: 1
      events:
        max_seconds: 300
        pre_capture: 5
        post_capture: 5
        retain:
          objects:
            default: 14
    rtmp:
      enabled: True
    objects:
      track:
        - person
        - motorcycle
        - bicycle
        - car
    snapshots:
      enabled: True
      timestamp: false
      bounding_box: True
      retain:
        default: 7
    motion:
      mask:
      - 1920,1080,1920,466,1800,444,1653,439,1488,469,1376,565,1201,605,1029,817,975,1080
      - 490,1080,148,1080,153,962,127,831,148,648,178,578,352,541,411,567,504,651,550,623,668,647,925,838,750,1080
      - 0,413,0,0,1920,0,1920,254,1920,322,1204,285,879,275,702,297,555,268,408,275,223,328
    zones:
      sondal_eversdal:
        coordinates: 331,300,496,253,699,268,1061,278,1531,291,1920,328,1920,470,1742,435,1604,424,1519,434,1442,502,1260,609,1077,839,1010,851,934,839,794,804,636,743,517,770,218,711,10,818,0,396
```

### Relevant log output

```shell
Some of the errors I get when trying the input_args: on the cameras that don't work.

frigate    | [s6-init] making user provided files available at /var/run/s6/etc...exited 0.
frigate    | [s6-init] ensuring user provided files have correct perms...exited 0.
frigate    | [fix-attrs.d] applying ownership & permissions fixes...
frigate    | [fix-attrs.d] done.
frigate    | [cont-init.d] executing container initialization scripts...
frigate    | [cont-init.d] done.
frigate    | [services.d] starting services
frigate    | [services.d] done.
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Starting Frigate (0.9.4-26ae608)
frigate    | Starting migrations
frigate    | [2021-12-03 23:21:49] peewee_migrate                 INFO    : Starting migrations
frigate    | There is nothing to migrate
frigate    | [2021-12-03 23:21:49] peewee_migrate                 INFO    : There is nothing to migrate
frigate    | [2021-12-03 23:21:49] frigate.mqtt                   INFO    : MQTT connected
frigate    | [2021-12-03 23:21:49] detector.coral                 INFO    : Starting detection process: 221
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Output process started: 222
frigate    | [2021-12-03 23:21:49] ws4py                          INFO    : Using epoll
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Camera processor started for 34steppingstones: 226
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Camera processor started for Klaasenbosch: 230
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Camera processor started for Sondal&Eversdal: 233
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Camera processor started for Vygeboom&Fairtrees: 235
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Camera processor started for Isabella&Fairtrees: 236
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Camera processor started for 9Gillian_Greenbelt: 238
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Camera processor started for Kendal&Huguenot: 239
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Camera processor started for Eversdal&Chantecler: 241
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Camera processor started for DawnCl&Lorelei: 243
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Camera processor started for 23Noeline_Greenbelt: 246
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Camera processor started for MountainView&Amanda: 247
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Camera processor started for Vygeboom&DurbanRD: 249
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Camera processor started for Sondal&Kendal: 257
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Capture process started for 34steppingstones: 258
frigate    | [2021-12-03 23:21:49] frigate.app                    INFO    : Capture process started for Klaasenbosch: 260
frigate    | [2021-12-03 23:21:50] frigate.app                    INFO    : Capture process started for Sondal&Eversdal: 264
frigate    | [2021-12-03 23:21:50] frigate.app                    INFO    : Capture process started for Vygeboom&Fairtrees: 269
frigate    | [2021-12-03 23:21:50] frigate.app                    INFO    : Capture process started for Isabella&Fairtrees: 271
frigate    | [2021-12-03 23:21:50] frigate.app                    INFO    : Capture process started for 9Gillian_Greenbelt: 274
frigate    | [2021-12-03 23:21:50] frigate.app                    INFO    : Capture process started for Kendal&Huguenot: 279
frigate    | [2021-12-03 23:21:50] frigate.app                    INFO    : Capture process started for Eversdal&Chantecler: 283
frigate    | [2021-12-03 23:21:50] frigate.app                    INFO    : Capture process started for DawnCl&Lorelei: 288
frigate    | [2021-12-03 23:21:50] frigate.app                    INFO    : Capture process started for 23Noeline_Greenbelt: 290
frigate    | [2021-12-03 23:21:50] frigate.app                    INFO    : Capture process started for MountainView&Amanda: 295
frigate    | [2021-12-03 23:21:50] frigate.app                    INFO    : Capture process started for Vygeboom&DurbanRD: 301
frigate    | [2021-12-03 23:21:50] frigate.app                    INFO    : Capture process started for Sondal&Kendal: 303
frigate    | [2021-12-03 23:21:50] ws4py                          INFO    : Using epoll
frigate    | [2021-12-03 23:21:49] frigate.edgetpu                INFO    : Attempting to load TPU as usb
frigate    | [2021-12-03 23:21:53] frigate.edgetpu                INFO    : TPU found
frigate    | [2021-12-03 23:21:56] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:41192]
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : Vygeboom&Fairtrees: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : Vygeboom&Fairtrees: ffmpeg process is not running. exiting capture thread...
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : Isabella&Fairtrees: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : Isabella&Fairtrees: ffmpeg process is not running. exiting capture thread...
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : 9Gillian_Greenbelt: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : 9Gillian_Greenbelt: ffmpeg process is not running. exiting capture thread...
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : Kendal&Huguenot: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : Kendal&Huguenot: ffmpeg process is not running. exiting capture thread...
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : Sondal&Eversdal: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : Sondal&Eversdal: ffmpeg process is not running. exiting capture thread...
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : 23Noeline_Greenbelt: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : 23Noeline_Greenbelt: ffmpeg process is not running. exiting capture thread...
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : Eversdal&Chantecler: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : Eversdal&Chantecler: ffmpeg process is not running. exiting capture thread...
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : Sondal&Kendal: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : Sondal&Kendal: ffmpeg process is not running. exiting capture thread...
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : DawnCl&Lorelei: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : DawnCl&Lorelei: ffmpeg process is not running. exiting capture thread...
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : Vygeboom&DurbanRD: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : Vygeboom&DurbanRD: ffmpeg process is not running. exiting capture thread...
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : 34steppingstones: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate    | [2021-12-03 23:22:00] frigate.video                  INFO    : 34steppingstones: ffmpeg process is not running. exiting capture thread...
^[[A^[[Bfrigate    | [2021-12-03 23:22:10] watchdog.34steppingstones      ERROR   : FFMPEG process crashed unexpectedly for 34steppingstones.
frigate    | [2021-12-03 23:22:10] watchdog.34steppingstones      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
frigate    | [2021-12-03 23:22:10] watchdog.34steppingstones      ERROR   : You may have invalid args defined for this camera.
frigate    | [2021-12-03 23:22:10] ffmpeg.34steppingstones.detect ERROR   : [rtsp @ 0x556adcd5ab40] UDP timeout, retrying with TCP
frigate    | [2021-12-03 23:22:10] ffmpeg.34steppingstones.detect ERROR   : [rtsp @ 0x556adcd5ab40] method PAUSE failed: 455 Method Not Valid in This State
frigate    | [2021-12-03 23:22:10] ffmpeg.34steppingstones.detect ERROR   : [rtsp @ 0x556adcd5ab40] Could not find codec parameters for stream 0 (Video: h264, none): unspecified size
frigate    | [2021-12-03 23:22:10] ffmpeg.34steppingstones.detect ERROR   : Consider increasing the value for the 'analyzeduration' and 'probesize' options
frigate    | [2021-12-03 23:22:10] ffmpeg.34steppingstones.detect ERROR   : Output file #0 does not contain any stream
frigate    | [2021-12-03 23:22:10] watchdog.Vygeboom&Fairtrees    ERROR   : FFMPEG process crashed unexpectedly for Vygeboom&Fairtrees.
frigate    | [2021-12-03 23:22:10] watchdog.Vygeboom&Fairtrees    ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
frigate    | [2021-12-03 23:22:10] watchdog.Vygeboom&Fairtrees    ERROR   : You may have invalid args defined for this camera.
frigate    | [2021-12-03 23:22:10] ffmpeg.Vygeboom&Fairtrees.detect ERROR   : [rtsp @ 0x55e3342c6b40] UDP timeout, retrying with TCP
frigate    | [2021-12-03 23:22:10] ffmpeg.Vygeboom&Fairtrees.detect ERROR   : [rtsp @ 0x55e3342c6b40] method SETUP failed: 454 Session Not Found
frigate    | [2021-12-03 23:22:10] ffmpeg.Vygeboom&Fairtrees.detect ERROR   : [rtsp @ 0x55e3342c6b40] Could not find codec parameters for stream 0 (Video: h264, none): unspecified size
frigate    | [2021-12-03 23:22:10] ffmpeg.Vygeboom&Fairtrees.detect ERROR   : Consider increasing the value for the 'analyzeduration' and 'probesize' options
frigate    | [2021-12-03 23:22:10] ffmpeg.Vygeboom&Fairtrees.detect ERROR   : Output file #0 does not contain any stream
frigate    | [2021-12-03 23:22:10] watchdog.Isabella&Fairtrees    ERROR   : FFMPEG process crashed unexpectedly for Isabella&Fairtrees.
frigate    | [2021-12-03 23:22:10] watchdog.Isabella&Fairtrees    ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
frigate    | [2021-12-03 23:22:10] watchdog.Isabella&Fairtrees    ERROR   : You may have invalid args defined for this camera.
frigate    | [2021-12-03 23:22:10] ffmpeg.Isabella&Fairtrees.detect ERROR   : [rtsp @ 0x561928d83b40] UDP timeout, retrying with TCP
frigate    | [2021-12-03 23:22:10] ffmpeg.Isabella&Fairtrees.detect ERROR   : [rtsp @ 0x561928d83b40] method SETUP failed: 454 Session Not Found
frigate    | [2021-12-03 23:22:10] ffmpeg.Isabella&Fairtrees.detect ERROR   : [rtsp @ 0x561928d83b40] Could not find codec parameters for stream 0 (Video: h264, none): unspecified size
frigate    | [2021-12-03 23:22:10] ffmpeg.Isabella&Fairtrees.detect ERROR   : Consider increasing the value for the 'analyzeduration' and 'probesize' options
frigate    | [2021-12-03 23:22:10] ffmpeg.Isabella&Fairtrees.detect ERROR   : Output file #0 does not contain any stream
frigate    | [2021-12-03 23:22:10] watchdog.Kendal&Huguenot       ERROR   : FFMPEG process crashed unexpectedly for Kendal&Huguenot.
frigate    | [2021-12-03 23:22:10] watchdog.Kendal&Huguenot       ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
frigate    | [2021-12-03 23:22:10] watchdog.Kendal&Huguenot       ERROR   : You may have invalid args defined for this camera.
frigate    | [2021-12-03 23:22:10] ffmpeg.Kendal&Huguenot.detect  ERROR   : [rtsp @ 0x55e503e6bb40] UDP timeout, retrying with TCP
frigate    | [2021-12-03 23:22:10] ffmpeg.Kendal&Huguenot.detect  ERROR   : [rtsp @ 0x55e503e6bb40] method SETUP failed: 454 Session Not Found
frigate    | [2021-12-03 23:22:10] ffmpeg.Kendal&Huguenot.detect  ERROR   : [rtsp @ 0x55e503e6bb40] Could not find codec parameters for stream 0 (Video: h264, none): unspecified size
frigate    | [2021-12-03 23:22:10] ffmpeg.Kendal&Huguenot.detect  ERROR   : Consider increasing the value for the 'analyzeduration' and 'probesize' options
frigate    | [2021-12-03 23:22:10] ffmpeg.Kendal&Huguenot.detect  ERROR   : Output file #0 does not contain any stream
frigate    | [2021-12-03 23:22:10] watchdog.23Noeline_Greenbelt   ERROR   : FFMPEG process crashed unexpectedly for 23Noeline_Greenbelt.
frigate    | [2021-12-03 23:22:10] watchdog.23Noeline_Greenbelt   ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
frigate    | [2021-12-03 23:22:10] watchdog.23Noeline_Greenbelt   ERROR   : You may have invalid args defined for this camera.
frigate    | [2021-12-03 23:22:10] watchdog.9Gillian_Greenbelt    ERROR   : FFMPEG process crashed unexpectedly for 9Gillian_Greenbelt.
frigate    | [2021-12-03 23:22:10] watchdog.9Gillian_Greenbelt    ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
frigate    | [2021-12-03 23:22:10] watchdog.9Gillian_Greenbelt    ERROR   : You may have invalid args defined for this camera.
frigate    | [2021-12-03 23:22:10] ffmpeg.9Gillian_Greenbelt.detect ERROR   : [rtsp @ 0x55f004ae8b40] UDP timeout, retrying with TCP
frigate    | [2021-12-03 23:22:10] ffmpeg.9Gillian_Greenbelt.detect ERROR   : [rtsp @ 0x55f004ae8b40] method SETUP failed: 454 Session Not Found
frigate    | [2021-12-03 23:22:10] ffmpeg.9Gillian_Greenbelt.detect ERROR   : [rtsp @ 0x55f004ae8b40] Could not find codec parameters for stream 0 (Video: h264, none): unspecified size
frigate    | [2021-12-03 23:22:10] ffmpeg.9Gillian_Greenbelt.detect ERROR   : Consider increasing the value for the 'analyzeduration' and 'probesize' options
frigate    | [2021-12-03 23:22:10] ffmpeg.9Gillian_Greenbelt.detect ERROR   : Output file #0 does not contain any stream
frigate    | [2021-12-03 23:22:10] watchdog.Sondal&Eversdal       ERROR   : FFMPEG process crashed unexpectedly for Sondal&Eversdal.
frigate    | [2021-12-03 23:22:10] watchdog.Sondal&Eversdal       ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
frigate    | [2021-12-03 23:22:10] watchdog.Sondal&Eversdal       ERROR   : You may have invalid args defined for this camera.
frigate    | [2021-12-03 23:22:10] ffmpeg.Sondal&Eversdal.detect  ERROR   : [rtsp @ 0x5643ff7873c0] UDP timeout, retrying with TCP
frigate    | [2021-12-03 23:22:10] ffmpeg.Sondal&Eversdal.detect  ERROR   : [rtsp @ 0x5643ff7873c0] method SETUP failed: 454 Session Not Found
frigate    | [2021-12-03 23:22:10] ffmpeg.Sondal&Eversdal.detect  ERROR   : [rtsp @ 0x5643ff7873c0] Could not find codec parameters for stream 0 (Video: h264, none): unspecified size
frigate    | [2021-12-03 23:22:10] ffmpeg.Sondal&Eversdal.detect  ERROR   : Consider increasing the value for the 'analyzeduration' and 'probesize' options
frigate    | [2021-12-03 23:22:10] ffmpeg.Sondal&Eversdal.detect  ERROR   : [NULL @ 0x5643ff7c9d80] Unable to find a suitable output format for 'pipe:'
frigate    | [2021-12-03 23:22:10] ffmpeg.Sondal&Eversdal.detect  ERROR   : pipe:: Invalid argument
frigate    | [2021-12-03 23:22:10] ffmpeg.23Noeline_Greenbelt.detect ERROR   : [rtsp @ 0x55e8ebb92b40] UDP timeout, retrying with TCP
frigate    | [2021-12-03 23:22:10] ffmpeg.23Noeline_Greenbelt.detect ERROR   : [rtsp @ 0x55e8ebb92b40] method SETUP failed: 454 Session Not Found
frigate    | [2021-12-03 23:22:10] ffmpeg.23Noeline_Greenbelt.detect ERROR   : [rtsp @ 0x55e8ebb92b40] Could not find codec parameters for stream 0 (Video: h264, none): unspecified size
frigate    | [2021-12-03 23:22:10] ffmpeg.23Noeline_Greenbelt.detect ERROR   : Consider increasing the value for the 'analyzeduration' and 'probesize' options
frigate    | [2021-12-03 23:22:10] ffmpeg.23Noeline_Greenbelt.detect ERROR   : Output file #0 does not contain any stream
frigate    | [2021-12-03 23:22:10] watchdog.Eversdal&Chantecler   ERROR   : FFMPEG process crashed unexpectedly for Eversdal&Chantecler.
frigate    | [2021-12-03 23:22:10] watchdog.DawnCl&Lorelei        ERROR   : FFMPEG process crashed unexpectedly for DawnCl&Lorelei.
frigate    | [2021-12-03 23:22:10] watchdog.DawnCl&Lorelei        ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
frigate    | [2021-12-03 23:22:10] watchdog.DawnCl&Lorelei        ERROR   : You may have invalid args defined for this camera.
frigate    | [2021-12-03 23:22:10] ffmpeg.DawnCl&Lorelei.detect   ERROR   : [rtsp @ 0x5604a2cc1b40] UDP timeout, retrying with TCP
frigate    | [2021-12-03 23:22:10] ffmpeg.DawnCl&Lorelei.detect   ERROR   : [rtsp @ 0x5604a2cc1b40] method SETUP failed: 454 Session Not Found
frigate    | [2021-12-03 23:22:10] ffmpeg.DawnCl&Lorelei.detect   ERROR   : [rtsp @ 0x5604a2cc1b40] Could not find codec parameters for stream 0 (Video: h264, none): unspecified size
frigate    | [2021-12-03 23:22:10] ffmpeg.DawnCl&Lorelei.detect   ERROR   : Consider increasing the value for the 'analyzeduration' and 'probesize' options
frigate    | [2021-12-03 23:22:10] ffmpeg.DawnCl&Lorelei.detect   ERROR   : Output file #0 does not contain any stream
frigate    | [2021-12-03 23:22:10] watchdog.Eversdal&Chantecler   ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
frigate    | [2021-12-03 23:22:10] watchdog.Eversdal&Chantecler   ERROR   : You may have invalid args defined for this camera.
frigate    | [2021-12-03 23:22:10] ffmpeg.Eversdal&Chantecler.detect ERROR   : [rtsp @ 0x5596a7957b40] UDP timeout, retrying with TCP
frigate    | [2021-12-03 23:22:10] ffmpeg.Eversdal&Chantecler.detect ERROR   : [rtsp @ 0x5596a7957b40] method SETUP failed: 454 Session Not Found
frigate    | [2021-12-03 23:22:10] ffmpeg.Eversdal&Chantecler.detect ERROR   : [rtsp @ 0x5596a7957b40] Could not find codec parameters for stream 0 (Video: h264, none): unspecified size
frigate    | [2021-12-03 23:22:10] ffmpeg.Eversdal&Chantecler.detect ERROR   : Consider increasing the value for the 'analyzeduration' and 'probesize' options
frigate    | [2021-12-03 23:22:10] ffmpeg.Eversdal&Chantecler.detect ERROR   : Output file #0 does not contain any stream
frigate    | [2021-12-03 23:22:10] watchdog.Vygeboom&DurbanRD     ERROR   : FFMPEG process crashed unexpectedly for Vygeboom&DurbanRD.
frigate    | [2021-12-03 23:22:10] watchdog.Vygeboom&DurbanRD     ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
frigate    | [2021-12-03 23:22:10] watchdog.Vygeboom&DurbanRD     ERROR   : You may have invalid args defined for this camera.
frigate    | [2021-12-03 23:22:10] ffmpeg.Vygeboom&DurbanRD.detect ERROR   : [rtsp @ 0x556fe8501b40] UDP timeout, retrying with TCP
frigate    | [2021-12-03 23:22:10] ffmpeg.Vygeboom&DurbanRD.detect ERROR   : [rtsp @ 0x556fe8501b40] method SETUP failed: 454 Session Not Found
frigate    | [2021-12-03 23:22:10] ffmpeg.Vygeboom&DurbanRD.detect ERROR   : [rtsp @ 0x556fe8501b40] Could not find codec parameters for stream 0 (Video: h264, none): unspecified size
frigate    | [2021-12-03 23:22:10] ffmpeg.Vygeboom&DurbanRD.detect ERROR   : Consider increasing the value for the 'analyzeduration' and 'probesize' options
```

### FFprobe output from your camera

```shell
Not sure how to run FFprobe
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

hikvision, zkteco

**Comment by spikeygg** [comment]
@Sarelseekat,

This is not a limitation of Frigate, it's a limitation of the Nvidia driver. Nvidia imposes a very tight limit of decode streams and encode streams to crunch simultaneously. If you want to crunch all the video from several cameras simultaneously using your GPU (like I did) you need to remove the GPU driver limitation. Take a look at this [keylase/nvidia-patch](https://github.com/keylase/nvidia-patch) driver patch for the details.

I bought this GPU specifically to host streams via plex and crunch video from security cameras. When I ran into this limitation I was floored. Thank keylase for putting that patch out there! Once you get it working you can use `nvidia-smi dmon -s u` to verify that you are actually crunching more streams at once. Here's some snippets from my machine running a Frigate docker with 7 ReoLink cameras running HD resolutions (like 2048x1536) at 10fps crunched via the GPU, using the patch:

`$ nvidia-smi dmon -s u
# gpu    sm   mem   enc   dec
# Idx     %     %     %     %
    0    15     4    18    14
    0    14     3    14    15
    0    12     3    18    12
    0    14     3    16    16
    0     9     2    13     9
    0    17     4    21    21
    0    10     2    12     9
    0    14     3    15    15
    0    14     3    16    14
    0    14     3    13    18
    0    12     3    16    11
    0    15     4    17    17
    0    12     3    13    12
    0    16     4    21    19
    0    10     2    12     9
    0    15     4    21    19
    0    13     3    18    14
    0    18     4    21    18
    0    10     2    13     9
^C
$ nvidia-smi
Sat Dec  4 10:43:34 2021
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 470.63.01    Driver Version: 470.63.01    CUDA Version: 11.4     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  NVIDIA GeForce ...  Off  | 00000000:03:00.0 Off |                  N/A |
| 34%   60C    P2    47W / 170W |   4200MiB / 12051MiB |     13%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+

+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|    0   N/A  N/A      2183      G   /usr/lib/xorg/Xorg                 67MiB |
|    0   N/A  N/A      4905      G   /usr/bin/gnome-shell                8MiB |
|    0   N/A  N/A   1833472      C   ffmpeg                            586MiB |
|    0   N/A  N/A   1833475      C   ffmpeg                            586MiB |
|    0   N/A  N/A   1833477      C   ffmpeg                            586MiB |
|    0   N/A  N/A   1833526      C   ffmpeg                            590MiB |
|    0   N/A  N/A   1833529      C   ffmpeg                            590MiB |
|    0   N/A  N/A   1845581      C   ffmpeg                            586MiB |
|    0   N/A  N/A   2067396      C   ffmpeg                            586MiB |
+-----------------------------------------------------------------------------+
$`

I could probably run a faster framerate but I don't want to overrun the coral.. so I limited the ReoLink within the firmware to 10fps. The system works well.

**Comment by Sarelseekat** [comment]
@spikeygg 

Wow. That sounds proper. I feel stumped at this moment, but I haven't lost hope. What OS are you running if I may ask? I've re-installed Ubuntu about 4 times in the last 2 days because of it failing to boot after loading Nvidia drivers. Super frustrating. I've also tried Debian 11 and Manjaro, but they just frustrate me more.

I have Ubuntu running again and saw posts where many people complain about Nvidia drivers causing Ubuntu to crash.
I also noticed a mention of driver versions in the Keylase repository instructions.

Will let you know if this solves my issues.

Thanks again for sharing your knowledge.

**Comment by spikeygg** [comment]
@Sarelseekat, I'm on Ubuntu 20.04.3 LTS. I haven't had any problems with my Nvidia drivers but because I'm using the GPU exclusively for remote stream and video crunch the system is effectively headless. If something wigged out on the local display I'd never know. The system is incredibly stable, I just checked the uptime and it's 46 days at the moment, it was probably the last time I did periodic updates and manually triggered a reboot.

Good luck!
